runtime/wasm-bpf-rs/src/state.rs

```diff
@@ -39,6 +39,10 @@ pub struct AppState {
     pub callback_func_name: String,
     pub wrapper_called: bool,
 }
+
+// We need a seprate thread to monitor the running process of a wasm program, so we must ensure that AppState to be Send
+#[cfg(test)]
+unsafe impl Send for AppState {}
```

Comment:
Why this cannot be safe?

Reply:
maybe you can wrapper within another struct with arc mutex...etc?

Reply:
Fixed. this unsafe impl is no longer needed